Packs/Silverfort/Integrations/Silverfort/README.md

```diff
@@ -0,0 +1,211 @@
+## Overview
+---
+Silverfort protects organizations from data breaches by delivering strong authentication across entire corporate networks and cloud environments, without requiring any modifications to endpoints or servers. Using patent-pending technology, Silverfort's agentless approach enables multi-factor authentication and AI-driven adaptive authentication even for systems that don’t support it today, including proprietary systems, critical infrastructure, shared folders, IoT devices, and more.
+<br>Use Silverfort integration to get & update Silverfort risk severity.
+<br>This integration was integrated and tested with Silverfort version 12.2.
```

Comment:
i think the digits flipped, should be 2.12

Reply:
Fixed